### ISSUE TITLE
Improve visibility of composer

### ISSUE THREAD

**Issue by nadonomy** [vector-im/element-web#17362]
> We've seen in a user test session someone squinting and saying they couldn't quite see the composer to be able to send a message.

Related issue: https://github.com/vector-im/internal-planning/issues/24

**Comment by SimonBrandner** [comment]
This is partially related to [#16304](https://github.com/vector-im/element-web/issues/16304). The unprofessional mock up linked there _could_ be an option

**Comment by niquewoodhouse** [comment]
**Using a 1px border of groupFilterPanel-bg-color**

![image](https://user-images.githubusercontent.com/1587166/119696925-982d6780-be47-11eb-97e1-66b507205103.png)

![image](https://user-images.githubusercontent.com/1587166/119697007-af6c5500-be47-11eb-8b58-02ada8bd4eb2.png)

**Using a 1px border of input-border-color on the timeline with groupFilterPanel-bg-color on the room header**

![image](https://user-images.githubusercontent.com/1587166/119697227-e3477a80-be47-11eb-84c4-5135a63e167f.png)

![image](https://user-images.githubusercontent.com/1587166/119697262-ed697900-be47-11eb-895b-eb81e73dcc1e.png)


Part of the reason people maybe aren't able to find the composer is the use of avatars to the left of the composer, while they're scanning for an input field.

I'm a bit nervous to make these kind of changes, because they're bigger than papercuts.


![image](https://user-images.githubusercontent.com/1587166/119697830-87312600-be48-11eb-9253-b4bd8fa14b71.png)


![image](https://user-images.githubusercontent.com/1587166/119697894-97e19c00-be48-11eb-950b-c96b69d732ac.png)

**Comment by t3chguy** [comment]
@niquewoodhouse all above options are feasible, the composer previously had such a border and it was removed by Nad iirc, so would be good to double-check the intent there

**Comment by niquewoodhouse** [comment]
> ...the composer previously had such a border and it was removed by Nad iirc, so would be good to double-check the intent there

@nadonomy would you mind briefly confirming why the border was removed, if it's ok to add one again?

I'd also be interested in hearing if we may remove the avatar next to the composer - to match the latest composer work on mobile.

Thanks

**Comment by reivilibre** [comment]
![image](https://user-images.githubusercontent.com/38398653/125827173-9c914981-978c-4520-9f84-6cf476ab355f.png)

I think this is basically the same issue but for different reasons, but I struggle to distinguish actual messages from the composer sometimes. It's usually when I've been thinking about what I'm writing, drop it for a second and come back, I can't tell which part of the text belongs to a sent message and which belongs to the composer.

Presumably the above ideas will sort this out as well. :)

**Comment by inducer** [comment]
An additional concern is that Firefox's link URL overlay can cover up the composer:

![grafik](https://user-images.githubusercontent.com/352067/158852180-e03f1df4-95a6-4130-8be1-74920ac0f671.png)

If the composer were spaced away from the bottom of the window (e.g. like Slack does this), this would not be an issue.

**Comment by HarHarLinks** [comment]
I want to bring to attention the point that I want to be able to see on a glance if I have an empty line at the start of the composer input. Here is an image with a horizontal line added via devtools similar to the screenshot above to illustrate the problem
![](https://user-images.githubusercontent.com/2803622/161119825-fe18acbd-2d7e-45b9-981f-6f606c3b8f9a.png)
<details><summary>Can you tell?</summary>There is in fact an empty line in the screenshot.</details>

**Comment by t3chguy** [comment]
This got reverted https://github.com/matrix-org/matrix-react-sdk/pull/8840

**Comment by t3chguy** [comment]
That needs to also get reverted

**Comment by t3chguy** [comment]
@HarHarLinks are you running the reverted PR Netlify? The revert hasn't landed yet

<img width="1393" alt="image" src="https://user-images.githubusercontent.com/2403652/173630470-ec448f95-c5fc-4b28-8e24-0ab0f4a31245.png">

Cannot reproduce

**Comment by t3chguy** [comment]
@HarHarLinks yup that is https://github.com/vector-im/element-web/issues/22459

--

All of the below apply to the now-reverted-change

https://github.com/vector-im/element-web/issues/22536
https://github.com/vector-im/element-web/issues/22508
https://github.com/vector-im/element-web/issues/22534
https://github.com/vector-im/element-web/issues/22506
https://github.com/vector-im/element-web/issues/22467
https://github.com/vector-im/element-web/issues/22459

**Comment by gsouquet** [comment]
Closing to make https://github.com/vector-im/element-web/issues/22072 the canonical issue
